src/models/index.ts

```diff
@@ -1,7 +1,7 @@
 import mongoose from 'mongoose';
 import 'dotenv/config';
 
-const { NODE_ENV, DATABASE_URL, DATABASE_URL_TEST, DATABASE_URL_PRODUCTION } = process.env;
+const { NODE_ENV = '', DATABASE_URL, DATABASE_URL_TEST, DATABASE_URL_PRODUCTION } = process.env;
```

Comment:
I am not getting what the ='' is doing here..
? can you please clarifiry? @alainburindi

Reply:
I'm having issues with environment variables, they are undefined at runtime

Reply:
Rebase from develop and check , we are setting them on runtime

Reply:
<img width="683" alt="Screen Shot 2020-05-02 at 18 38 47" src="https://user-images.githubusercontent.com/18573157/80870040-38947500-8ca4-11ea-8deb-ba29eeb3aaaa.png">
Check this.....

Reply:
The way we are setting them up...

Reply:
I have the same setup. the problem I was having was on runtime. TypeScript couldn't compile, it was saying expecting string but undefined was given


![scri](https://user-images.githubusercontent.com/41288911/80910737-c23e5400-8d31-11ea-94da-185d05152036.png)

Reply:
Are you sure we you have a `.env` file in your project folder?
What did you get when you do console.log(process.env)?

Reply:
@alainburindi !!!

Reply:
@alainburindi You can update the global interface